karma.conf.js

```diff
@@ -18,8 +18,8 @@ module.exports = (config) => {
       'https://ajax.googleapis.com/ajax/libs/angularjs/1.6.1/angular.min.js',
       'https://code.angularjs.org/1.6.1/angular-route.min.js',
       'public/js/helper.js',
```

Comment:
okay, for files in the same directory you can use pattern matching to load them. eg. all files in './public' which have the js extension can be loaded like this './public/*.js'